templates/AureliaSpa/ClientApp/boot.ts

```diff
@@ -0,0 +1,15 @@
+import {Aurelia} from 'aurelia-framework';
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap';
+ 
+// comment out if you don't want a Promise polyfill (remove also from webpack.config.js)
+import * as Bluebird from  'bluebird';
```

Comment:
Layout issues in this file (e.g., double-whitespace on line 1, uses two-char indentation elsewhere when we use four-char in all the other templates).